site/manifest.json

```diff
@@ -8,6 +9,51 @@
     "1.6": "2020-11-21"
   },
   "istio_distributions": [
+    {
+      "version": "1.12.1",
+      "flavor": "tetrate",
+      "flavor_version": 1,
```

Comment:
1.12.1 - Did we have a v0 release already?

Reply:
@deva26 a [v0](https://github.com/tetratelabs/istio/tree/test-1.12.1-tetrate-v0) was tagged but never built. A [v1](https://github.com/tetratelabs/istio/tree/test-1.12.1-tetrate-v1) was tagged which I used to build the images.